common/gen_bash_afs_funcs

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash
```

Comment:
This is very specific to IBM.  Really shouldn't be something we need up here.  Can we separate it out to be internal use only?